Dockerfile

```diff
@@ -25,32 +25,50 @@
 #     -i -t --rm \
 #     --runtime=nvidia \
 #     core3d/danesfield \
-#     danesfield/tools/material_classifier.py --cuda ...
+#     danesfield/tools/material_classifier.py --cuda ...    
+
+# Download and install miniconda3
+# Based on https://github.com/ContinuumIO/docker-images/blob/fd4cd9b/miniconda3/Dockerfile
+# The step to update 'conda' is necessary to avoid the following error when
+# downloading packages (see https://github.com/conda/conda/issues/6811):
+#
+#     IsADirectoryError(21, 'Is a directory')
+#
```

Comment:
What's the context / purpose of this comment block?  It's not clear to me from the surrounding code (maybe my memory is fuzzy here)

Reply:
It's in the wrong place. It should be above the RUN command in line 72. Will be fixed in next commit